compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1890,7 +1898,10 @@ impl<'tcx> TyCtxt<'tcx> {
     }
 
     pub fn find_field_index(self, ident: Ident, variant: &VariantDef) -> Option<usize> {
-        variant.fields.iter().position(|field| self.hygienic_eq(ident, field.ident, variant.def_id))
+        variant
+            .fields
+            .iter()
+            .position(|field| self.hygienic_eq(ident, field.ident(self), variant.def_id))
```

Comment:
Should `field.ident(self)` be lifted out of the loop?

Reply:
This cannot be lifted, since we have a different `Ident` for each field.